src/Integration.Vsix/ErrorList/SinkManager.cs

```diff
@@ -25,10 +25,11 @@
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
     // Interface introduced to simplify testing.
-    internal interface ISinkManagerRegister
```

Comment:
I moved the `ISinkManagerRegister` interface to the new `SonarErrorLIstDataSource` class and added `ISinkManager`.